src/buildTree.js

```diff
@@ -52,8 +52,8 @@ const reactTreeToFlexTree = (
     for (let index = 0; index < children.length; index += 1) {
       const childComponent = children[index];
       const childStyles =
-        childComponent.props && childComponent.props.styles
-          ? childComponent.props.styles
+        childComponent.props && childComponent.props.style
```

Comment:
Due to the typo the `childStyles` always have been an empty object rather than the actual style. As a result the `decrementIndex` condition couldn't detect absolute positioned layers.